.gitlab-ci.yml

```diff
@@ -1056,6 +1066,35 @@ workflow:
     when: on_success
   - when: manual
 
+.çç:
```

Comment:
What is this?

Reply:
I had some troubles with a merge that I was fixing right now, this is part of that it seems, sorry!